src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -194,6 +194,13 @@ configurationRegistry.registerConfiguration({
 				'panel': 'always',
 			}
 		},
+		'chat.runtimeSessionContext.maxExecutionHistoryCost': {
+			type: 'number',
+			tags: ['experimental'],
+			description: nls.localize('chat.runtimeSessionContext.maxExecutionHistoryCost', "The maximum token cost for the runtime session execution history context. This is used to limit the size of the context that is sent to the model. The value is in tokens, with a default of 8192 tokens."),
```

Comment:
This setting's name and value still need to be updated to state that it limits characters, not tokens. We can't compute the token cost ahead of time b/c the tokenization varies per model.